app/models/council.rb

```diff
@@ -1,18 +1,34 @@
 # encoding: UTF-8
 class Council < ActiveRecord::Base
+
+  # Associations
   has_one :page, dependent: :destroy
   has_many :posts
   has_many :profiles, through: :posts
-  validates :title,:url,:presence => true
+  has_and_belongs_to_many :cafe_works
+
+  # Validation
+  validates :title, :url, presence: true
   validates :url, uniqueness: true
+
   def to_s
     self.title
   end
+
+  # To use the url as actual url
   def to_param
-    if (self.url) 
+    if (self.url)
```

Comment:
Don't use parentheses around the condition of an `if`.<br>Redundant `self` detected.